src/state/therapies/actions.ts

```diff
@@ -32,13 +33,17 @@ export const createTherapy =
       (payload) => {
         dispatch({
           type: CREATE_THERAPY_SUCCESS,
-          payload,
+          payload: {
+            ...payload,
+            startDate: format(new Date(+payload.startDate!), "dd/MM/yyyy"),
+            endDate: format(new Date(+payload.endDate!), "dd/MM/yyyy"),
```

Comment:
I think that should be more practical check if the payload.startDate or payload.endDate both exists before convert them into a Date.

Reply:
Maybe I miss the point but when or where are you gonna use startDate and endDate?

Reply:
The created data will be used to update the therapy table, instead of refreshing the whole table each time a new data is created.